Foundation/NSCharacterSet.swift

```diff
@@ -168,6 +168,10 @@ public class NSCharacterSet : NSObject, NSCopying, NSMutableCopying, NSCoding {
         return CFCharacterSetCreateCopy(kCFAllocatorSystemDefault, self._cfObject)
     }
     
+    public override func mutableCopy() -> AnyObject {
```

Comment:
I think there seems to be a failure in all as? testing protocol adoption on linux. It might be useful to help narrow that one down with some sort of reduced test case for the stdlib/compiler team to take a look at because this will mean that we have a number of nasty runtime lurking failures scattered all over the place that we have not yet found since we don't have full coverage yet.